src/pages/SetPassword.vue

```diff
@@ -23,16 +23,14 @@
         type="password"
       />
       <ae-label
-        :for="_uid"
-        :help-text="errors.first('password_repeat')"
+        :for="_uid2"
+        :help-text="errors.first('passwordRepeat')"
         help-type="dramatic"
       >Confirm your password</ae-label>
       <ae-input
         v-validate="'required|confirmed:'+password"
-        v-focus="true"
-        :id="_uid"
-        v-model="password_repeat"
-        name="password_repeat"
+        :id="_uid2"
```

Comment:
`_uid` is unique ID of Vue component instance it is used to build IDs of DOM nodes because they also should be unique among IDs used in this application. I'm not sure that `_uid2` is defined somehow. In case if components have several fields I'm building ID of DOM node as a template that depends on `_uid`, like [this one](https://github.com/aeternity/aepp-identity/blob/beb222bd7a31bc6766c8b06b6c633b62809e44c4/src/pages/Transfer/Transfer.vue#L42).